gradle.properties

```diff
@@ -1 +1 @@
-version=0.2.1-SNAPSHOT
+version=0.3.0-SNAPSHOT
```

Comment:
is there a versioning policy ? do we change the second digit of the version each time there is a new native function ?

Reply:
SemVer.  New Long-term APIs should be minor.  https://semver.org/